commonblocks/static/commonblocks/js/hallo-bootstrap.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/**
+ * registerHalloPlugin() applies to all editor instances of a Page.
+ * We want to encapsulate custom plugin configuration, per field.
```

Comment:
This comment would be more helpful if it was right above the "saving plugins" / "restoring plugins" lines, because the overall function does much more than that.